addon/templates/components/uni-header-menu.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if withExit}}
+    <button class="uni-header__nav__button" {{action "triggerExit"}}>{{exitLabel}}</button>
+{{else}}
+    {{#if loggedIn}}
+        {{uni-header-button button=headerButtons.[0]}}
+
+        {{#uni-header-dropdown options=headerOptions selected="Filipe" as |option componentName|}}
+            {{component componentName option=option}}
+        {{/uni-header-dropdown}}
+    {{else}}
+        {{#each headerButtons as |option|}}
+            {{uni-header-button button=option action="click"}}
```

Comment:
you have an action here, but you don't use it in the component uni-header-button